prod/manifest.json

```diff
@@ -1,36 +1,30 @@
 {
   "manifest_version": 2,
-
   "name": "codesy.io",
   "description": "codesy is a pay-what-you-want market for the open source community to encourage coders to fix important bugs.",
-  "version": "1.0",
-
+  "version": "2.0",
+  "bid_domain" : "api.codesy.io",
```

Comment:
This causes a warning in Chrome:
![codesy-plugin-warning](https://cloud.githubusercontent.com/assets/71928/5641519/f999b342-95ff-11e4-94f7-fdc1c23aacc7.png)
But it doesn't even show up if you're not in developer mode. Do we care?